components/brave_new_tab_ui/components/default/gridSites/index.ts

```diff
@@ -72,29 +72,53 @@ export const AddSiteTile = styled('button')<AddSiteTileProps>`
   }
 `
 
-export interface ListProps {
-  blockNumber: number
-}
-
-export const List = styled('div')<ListProps>`
-  justify-self: start;
-  align-items: normal;
+export const List = styled('div')`
   height: 100%;
+  justify-self: start;
+
+  // Add right padding of one column, so there's a nice gap between pages.
+  padding-right: var(--grid-column-width);
+  
   display: grid;
-  grid-template-columns: repeat(${p => Math.min(p.blockNumber, 6).toString()}, 86px);
+  align-items: normal;
   justify-content: var(--ntp-item-justify, start);
-  padding: 24px 56px 24px 24px;
+
+  grid-template-columns: repeat(var(--grid-columns), var(--grid-column-width));
+  grid-auto-flow: row;
+
+  scroll-snap-align: start;
+`
+
+export const PagesContainer = styled('div')`
+  max-width: 100vw;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  margin-bottom: 24px;
+
+  --grid-columns: 6;
+  --grid-column-width: 86px;
 
   @media screen and (max-width: 700px) {
-    grid-template-columns: repeat(${p => Math.min(p.blockNumber, 3).toString()}, 86px);
+    --grid-columns: 4;
```

Comment:
Note, I know these aren't the same values, but it looks a bit better (i.e. the number of columns more nearly fills the screen when the screen is small).

Before (700px ish):
![Screenshot from 2022-06-13 11-21-28](https://user-images.githubusercontent.com/7678024/173257805-347702e4-2b53-4fbd-89a2-a81eb5f1e93c.png)

After (700px ish):
![Screenshot from 2022-06-13 11-22-05](https://user-images.githubusercontent.com/7678024/173257813-2641c51b-4193-4226-b04d-3af1c18d57ac.png)


Before (360px ish):
![Screenshot from 2022-06-13 11-21-48](https://user-images.githubusercontent.com/7678024/173257823-7f2f7ae2-c73b-4310-9ebb-df52763bb46b.png)

After (360px ish):

![Screenshot from 2022-06-13 11-22-17](https://user-images.githubusercontent.com/7678024/173257837-15ffbe19-b17d-431e-bae8-25e0221a838e.png)

Reply:
looks great